tests/test_ir/test_key_value_pair_log_event.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+from typing import Any, Dict
+
+from test_ir.test_utils import JsonLinesFileReader, TestCLPBase
+
+from clp_ffi_py.ir import KeyValuePairLogEvent
+
+
+class TestCaseKeyValuePairLogEvent(TestCLPBase):
+    """
+    Class for testing `clp_ffi_py.ir.KeyValuePairLogEvent`.
+    """
+
+    test_data_dir: Path = Path("test_data")
+    jsonl_dir: Path = Path("jsonl")
```

Comment:
* I find it a bit odd that these are paths since they don't actually point at anything on disk until they're resolved against `current_dir`. Personally, I would just make them strings.
* If you do make them strings, how about calling them `test_data_dir_name` and `jsonl_test_data_dir_name`?
* If you don't make them strings, how about calling `jsonl_dir` -> `jsonl_test_data_dir`?

Reply:
how about:
```
jsonl_test_data_dir: Path = Path("test_data") / Path("jsonl")
```